installer/dev/tracelogging.h

```diff
@@ -30,6 +30,7 @@ void __stdcall wilResultLoggingCallback(const wil::FailureInfo& failure) noexcep
 
 class WindowsAppRuntimeInstaller_TraceLogger final : public wil::TraceLoggingProvider
 {
+    #pragma prefast(disable:6387, "PreFast gets confused over tracelogging parameters")
```

Comment:
I think "disable" disables the warning for the rest of the translation unit.  Does prefast support "suppress" here instead to just disable the warning for the next line?  Alternatively, can you do prefast(push) and (pop) like you can for warnings?  https://learn.microsoft.com/en-us/cpp/preprocessor/warning?view=msvc-170

Reply:
Yes, that's possible: [https://learn.microsoft.com/en-us/previous-versions/windows/embedded/gg155764(v=winembedded.70)#parameters](https://learn.microsoft.com/en-us/previous-versions/windows/embedded/gg155764(v=winembedded.70)#parameters)

Reply:
Although for the purpose this usage I think changing `disable` to `suppress` is sufficient since we only need it for the following statement.

Reply:
good idea. i didn't know one could do that.